egs2/rats/spk1/conf/train_RawNet3.yaml

```diff
@@ -0,0 +1,91 @@
+# RawNet3 reproduce recipe configuration.
+# Requires approx. 40GRAM per GPU when run with 2GPUs
+# or 20GRAM when run with 4GPUs.
```

Comment:
Dos this align with your actual training config? 
If not, can you reflect the setting you ran?